presto-docs/src/main/sphinx/release/release-306.rst

```diff
@@ -0,0 +1,76 @@
+===========
+Release 306
+===========
+
+General Changes
+---------------
+
+* Fix planning failure for queries containing a ``LIMIT`` after a global
+  aggregation. (:issue:`437`)
+* Fix missing column types in EXPLAIN output. (:issue:`328`)
+* Fix updating peak revocable memory reservation. (:issue:`413`)
+* Fix blocking on memory when revocable memory is used. (:issue:`413`)
+* Fix double memory accounting for spilling aggregations. (:issue:`413`)
+* Fix possible CPU active waiting when window operator spill is triggered. (:issue:`468`)
+* Fix incorrect view name displayed by ``SHOW CREATE VIEW``. (:issue:`433`)
+* Allow specifying ``NOT NULL`` when creating tables or adding columns. (:issue:`418`)
+* Add a config option (``query.stage-count-warning-threshold``) to specify a
+  per-query threshold for the number of stages. When this threshold is exceeded,
+  a ``TOO_MANY_STAGES`` warning is raised. (:issue:`330`)
+* Support session properties with comma or equality sign. (:issue:`407`)
+* Remove the ``deprecated.legacy-unnest-array-rows`` configuration option.
+  The legacy behavior for ``UNNEST`` of arrays containing ``ROW`` values is no
+  longer supported. (:issue:`430`)
+* Remove the ``deprecated.legacy-row-field-ordinal-access`` configuration option.
+  The legacy mechanism for accessing fields of anonymous ``ROW`` types is no longer
+  supported. (:issue:`428`)
+* Remove the ``deprecated.group-by-uses-equal`` configuration option. The legacy equality
+  semantics for ``GROUP BY`` are not longer supported. (:issue:`432`)
+* Remove the ``deprecated.legacy-map-subscript``. The legacy behavior for the map subscript
+  operator on missing keys is no longer supported. (:issue:`429`)
+* Remove the ``deprecated.legacy-char-to-varchar-coercion`` configuration option. The
+  legacy coercion rules between ``CHAR`` and ``VARCHAR`` types are no longer
+  supported. (:issue:`431`)
+* Remove deprecated ``distributed_join`` system property. Use ``join_distribution_type``
+  instead. (:issue:`452`)
+
+Hive Connector Changes
+----------------------
+
+* Fix procedure calls immediately after startup, before any other queries are run. (:issue:`414`)
```

Comment:
Reading this again, I'm thinking we should expand it:

> Fix calling procedures immediately after startup, before any other queries are run. Previously, the procedure call would fail and also cause all subsequent Hive queries to fail.